src/Components/MissionItems.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
```

Comment:
- Nice to pass your data by using props but I  would like to suggest you not disable linters errors but find solutions for them because Linters improve our clean code 😄 [OPTIONAL]
- Try to install packages for prop-types then make some variables required or non-required 👀  [check more](https://www.npmjs.com/package/prop-types)